curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f5d2776c854e069560fbe6.md

```diff
@@ -11,18 +11,39 @@ The first `fieldset` will hold name, email, and password fields. Start by adding
 
 # --hints--
 
-You should add four `label` elements.
+There should be three `fieldset` elements in total.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.length, 4);
+assert.equal(document.querySelectorAll('fieldset')?.length, 3);
 ```
 
-You should add the `label` elements as direct children to the first `fieldset`. Make sure to close the `label` elements.
+The `fieldset` elements should all be direct children of the `form` element.
+
+```js
+assert.equal(document.querySelectorAll('form > fieldset')?.length, 3);
+```
+
+The four `label` elements should all be direct children of the first `fieldset` element. Make sure you close the `label` elements.
```

Comment:
This test does not test this, and the test further down does.

```suggestion
The four `label` elements should all be direct children of the first `fieldset` element.
```

Reply:
It does test for it because if the labels are not closed correctly the browser will do it and the labels end up nested inside each other. That is the point of letting the camper know they may not have closed the labels correctly if the labels are not direct children of the fieldset.

Reply:
And to have it before the closing label tag count so as not to trigger that test incorrectly if the number of label elements is not 4. It is a confusing hint to get if your labels are closed correctly but you have too many or too few.